dhcp_server/DOCS.md

```diff
@@ -38,99 +38,126 @@ networks:
     range_end: 192.168.1.200
     broadcast: 192.168.1.255
     gateway: 192.168.1.1
-    interface: end0
+    interface: eth0
```

Comment:
Almost none of the OS installations by default use `eth0` these days. Someone even recently asked to change that https://github.com/home-assistant/addons/pull/3638 :sweat_smile: 

So let's leave it as `end0` now.